Clustered execution is supported (#682)
grafana/k6-docs: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md

For advanced ramping, you can use [scenarios](/using-k6/scenarios) and the `ramping-vus` executor.

## Execution modes

k6 supports three execution modes to run a k6 test: local, distributed, and cloud. 

> > One of the k6 design goals is to allow switching the execution modes with minimal changes. 

Reviewer: ```suggestion
> **The design of k6 lets users switch between execution modes with minimal changes.**
```

> &nbsp;
> 

> > The primary goal is to support **running the test in the various execution modes without modifying the k6 script**. This ability will enable reusing your tests - no matter the environment and context of your tests.

Reviewer: ```suggestion
> This lets users reuse tests&mdash;no matter the environment and context. Making tests portable across multiple modes of execution is a key design goal of k6.
```

> 1. [Local](#running-local-tests): the test execution is run entirely on a single machine, container, or CI server. 

Reviewer: ```suggestion
1. [Local](#running-local-tests): the test runs entirely on a single machine, container, or CI server. 
```

> 1. [Local](#running-local-tests): the test execution is run entirely on a single machine, container, or CI server. 

Reviewer: ```suggestion
1. [Local](#running-local-tests): the test execution happens entirely on a single machine, container, or CI server. 
```

  ```bash
  k6 run script.js
  ```

2. [Distributed](https://github.com/grafana/k6-operator): the test execution is distributed across a Kubernetes cluster. 
  

>   <CodeGroup labels={["Running", "k6-resource.yml"]} lineNumbers={[true]}> 

Reviewer: ```suggestion
  <CodeGroup labels={["Running", "k6-resource.yaml"]} lineNumbers={[true]}> 
```

  ```bash

>   kubectl apply -f /path/k6-resource.yml

Reviewer: ```suggestion
  kubectl apply -f /path/k6-resource.yaml
```

  ```

  ```yml
  ---
  apiVersion: k6.io/v1alpha1
  kind: K6
  metadata:
    name: k6-sample
  spec:
    parallelism: 4
    script:
      configMap:
        name: "k6-test"
        file: "script.js"
  ```

  </CodeGroup>

> 3. [Cloud](/cloud): the test execution is run on the k6 Cloud.  

Reviewer: ```suggestion
3. [Cloud](/cloud): the test execution happens on k6 Cloud.  
```

  ```bash
  k6 cloud script.js
  ```

>   Additionally, k6 Cloud can run cloud tests on your [own cloud infrastructure](/cloud/cloud-faq/private-load-zones/), and accepts the test results from a [local](/results-visualization/cloud/) or [distributed test](https://github.com/grafana/k6-operator#k6-cloud-output).

Reviewer: ```suggestion
  Additionally, k6 Cloud can run cloud tests on your [own cloud infrastructure](/cloud/cloud-faq/private-load-zones/), and accept the test results from a [local](/results-visualization/cloud/) or [distributed test](https://github.com/grafana/k6-operator#k6-cloud-output).
```
